server/game.js

```diff
@@ -109,23 +109,32 @@ export default class Game
 		{
 			// give each player their respective view of what everyone's hand looks like
 			let handsView = playersArray.map((player, i) => {
-				// your own hand
-				if(player.id == playersArray[p].id)
+				// your own hand and you're the dealer

```

Comment:
I think the comment in line 112 is the misleading part. The dealer and the deal button aren't the same thing, and neither is the game owner and the dealer, it's currently set up so only the owner can press 'deal', but this doesn't make them the dealer. I think only the game owner should be pressing 'deal', otherwise all players could press deal whenever they choose, which doesn't seem right. 

This isn't an issue until we have play-again functionality is it? And the dealer will go round sequentially or maybe based on the previous winner, but the game owner would still press 'deal'.

I'll amend the comment in this code to say owner and not dealer, happy to change other stuff if you spot a more fundamental issue with this approach?